mixins/form.jsx

```diff
@@ -96,7 +95,7 @@ module.exports = {
     this.updateHeight();
   },
   submit: function(action, props, callback) {
-    props.localeCode = this.state.localeCode || "US";
+    props.locale = this.props.locales[0];
```

Comment:
Isn't en-US what we want? At this point we're still fairly generic. I think "US" was just a mistake from the paypal locale, which, we should solve in a totally different way.